src/index.ts

```diff
@@ -1,6 +1,7 @@
 import * as bech32 from 'bech32';
 import * as bs58check from 'bs58check';
 import * as cashaddr from 'cashaddrjs';
+import * as ecc from 'eosjs-ecc';
```

Comment:
Please name this something specific to EOS.

Reply:
Changed to eos.

Reply:
Scatter

![scatter_eos](https://user-images.githubusercontent.com/49895199/69954893-1e901400-1505-11ea-85d1-3b6a31880e53.png)

Reply:
eos-Voter
![voter](https://user-images.githubusercontent.com/49895199/69955308-0ff62c80-1506-11ea-9bbb-ba1b00504029.png)

Reply:
https://eosauthority.com/

![eos_authority](https://user-images.githubusercontent.com/49895199/69955414-403dcb00-1506-11ea-83da-a3a0c905ebbc.png)